modules/imaging_browser/jsx/ImagePanel.js

```diff
@@ -39,13 +39,13 @@ var ImagePanelHeader = React.createClass({
                         Header Info
                         </button>
                     <span className="caret"></span>
-                </div>
-            </div>
+                </span>
+            </span>
             );
     return (
 
-            <div className="panel-heading">
-            <input type="checkbox" data-file-id={this.props.FileID}
+            <div className="panel-heading clearfix">
```

Comment:
I've never heard of a CSS "clearfix" being anything other than an IE6 fix, but okay.. if it's necessary it's fine. (Though this has conflicts again.)